.github/workflows/release.yml

```diff
@@ -25,8 +25,9 @@ jobs:
         npm run stats
         npm test
         [[ `git status --porcelain` ]] || exit
+
         git add .
-        git commit -am "update all-the-package-repos"
-        npm version minor -m "bump minor to %s"
+        npm version minor -f -m "update all-the-package-repos to %s"
```

Comment:
So this command somehow does a git commit?

Reply:
yep, it first bumps the version, and then commits and creates a tag to such commit.

> https://docs.npmjs.com/cli/version